example/src/main/scala/example/CSRF.scala

```diff
@@ -0,0 +1,14 @@
+package example
+
+import zhttp.http.Middleware.csrf
+import zhttp.http._
+import zhttp.service.Server
+import zio.{App, ExitCode, URIO}
+
+object CSRF extends App {
+  // Will give FORBIDDEN when CSRF token in header doesn't match CSRF token set in cookies
+  val app: HttpApp[Any, Nothing] = Http.ok @@ csrf("x-token", "token")
```

Comment:
```suggestion
  val app: HttpApp[Any, Nothing] = Http.ok @@ csrf(headerName = "x-token", cookieName= "token")
```
Something like this will avoid confusion

Reply:
I feel we should keep the same name for the header and cookie.

Reply:
Example should still be updated as amit mentioned.